offload_vio/device_tx/plugin.cpp

```diff
@@ -20,7 +20,8 @@
 #include <opencv2/core/mat.hpp>
 
 using namespace ILLIXR;
-#define USE_COMPRESSION
+
+// #define USE_COMPRESSION
```

Comment:
why defaulting to non-compression?

Reply:
Given Boyuan's comment, we probably should not make it a hard requirement that the user has gstreamer installed properly.

Reply:
ok